rllib/models/catalog.py

```diff
@@ -202,7 +201,6 @@
 # fmt: on
 
 
-@Deprecated(old="rllib.models.catalog.ModelCatalog", error=False)
```

Comment:
Same here, why are we removing this? Simply too verbose?

Reply:
Yes, too verbose and confusing.